itou/companies/admin.py

```diff
@@ -58,7 +58,10 @@ class FinancialAnnexesInline(ItouTabularInline):
     readonly_fields = ("number", "state", "start_at", "end_at", "is_active")
     can_delete = False
 
-    ordering = ("-number",)
+    ordering = (
+        "-end_at",
+        "-number",
+    )
```

Comment:
Pour avoir quelque chose d'un peu plus chronologique ?
```suggestion
    ordering = (
        "-end_at",
        "-start_at",
        "-number",
    )
```